tests/e2e/specs/stories-editor/story-templates.js

```diff
@@ -22,6 +29,66 @@ describe( 'Story Templates', () => {
 
 			await expect( page ).toMatchElement( '.block-editor-inserter__no-results' );
 		} );
+
+		describe( 'With non-tempale Reusable block', () => {
+			// Add Reusable block.
+			beforeAll( async () => {
+				await createNewPost();
+
+				const isTopToolbarEnabled = await page.$eval( '.edit-post-layout', ( layout ) => {
+					return layout.classList.contains( 'has-fixed-toolbar' );
+				} );
+				if ( ! isTopToolbarEnabled ) {
+					await clickOnMoreMenuItem( 'Top Toolbar' );
+				}
+
+				// Remove all blocks from the post so that we're working with a clean slate.
+				await page.evaluate( () => {
+					const blocks = wp.data.select( 'core/block-editor' ).getBlocks();
+					const clientIds = blocks.map( ( block ) => block.clientId );
+					wp.data.dispatch( 'core/block-editor' ).removeBlocks( clientIds );
+				} );
+
+				// Insert a paragraph block
+				await insertBlock( 'Paragraph' );
+				await page.keyboard.type( 'Reusable block!' );
+
+				await clickButtonByLabel( 'More options' );
+
+				const convertButton = await page.waitForXPath( '//button[text()="Add to Reusable Blocks"]' );
+				await convertButton.click();
+			} );
+
+			afterAll( async () => {
+				await visitAdminPage( 'edit.php', 'post_type=wp_block' );
```

Comment:
This could use a comment for what it does (removes all reusable blocks), will add it with other fixes from review.